contrib/dockerd-rootless.sh

```diff
@@ -128,5 +128,23 @@ else
 		mount --rbind ${realpath_etc_ssl} /etc/ssl
 	fi
 
-	exec dockerd $@
+	dockerd="$(which dockerd)"
```

Comment:
`which` is often missing

Reply:
[ ] [1. [ ] 1. [`Have```]([url]([url](url)))]([url](url))

Reply:
Unfortunately Delve requires a full path to the `dockerd` binary. I was reluctant to go with `which` as well, since it’s not a built-in bash function. However, I’ve checked Docker images of common distros like Ubuntu, Debian, Fedora, OpenSUSE, Arch, Alpine, even Busybox - all of them have `which` included. I assumed that the `hack/make.sh` scripts are meant to be executed from within the development container, so it’s safe to assume that `which` is available.

Reply:
> @kponichtera did `command -v` not work?

It does, I replaced `which` with `command -v` already. :ok_hand: